RxCocoa/Traits/ControlEvent.swift

```diff
@@ -30,9 +30,9 @@ public protocol ControlEventType : ObservableType {
 
     **It is the implementor’s responsibility to make sure that all other properties enumerated above are satisfied.**
 
-    **If they aren’t, using this trait will communicate wrong properties, and could potentially break someone’s code.**
+    **If they aren't, using this trait will communicate wrong properties, and could potentially break someone’s code.**
```

Comment:
not a typo

Reply:
Not a typo really, but maybe change this for consistency?
```
ag "aren’t" | wc -l // 1
ag "aren't" | wc -l // 7
```

Reply:
I don't mind, but I imagine there are many places that don't have magic quotes so it might be hard really getting all of them in a single PR. Feel free to make that change.

Reply:
Reverted for now, it can be made in another PR if needed